link.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from scrapy.linkextractors import LinkExtractor
+from scrapy.http import HtmlResponse, TextResponse, Response
+from six.moves.urllib.parse import urlparse
+import glob
+from timeit import default_timer as timer
+import click
+import io
+
+
+def main():
+    start = timer()
+
+    url = 'http://scrapinghub.com/'
+    link_extractor = LinkExtractor()
+    total = 0
+    for files in glob.glob('sites/index*'):
+
+        f = (io.open(files, "r", encoding="ISO-8859-1"))
```

Comment:
I had downloaded them by taking the Alexa top sites list and using `wget`. The problem is solved now, I used a python script : 
```
with codecs.open(files,'r',encoding='ISO-8859-1') as f:
        text = f.read()
        print (text)
            
    with io.open(files,'w',encoding='utf8') as f:
       f.write(text)
The code seems to run fine.
```